shard/connpool.lua

```diff
@@ -262,7 +304,7 @@ end
 
 -- function to check a connection after it's established
 local function check_connection(self, conn)
```

Comment:
Another function to check a state, which sometimes is used, and sometimes not. How about to add it to a server metatable?